### PR TITLE
Update AboutUs - Hyperlinked Image and Table

### PR DESCRIPTION
**Pull request by Ardentsoul**
Replaced the github and PPP hyperlinks with images and put roles and responsibilities into a table using github-markdown.

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -13,53 +13,53 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
...
+[<img src="images/github-icon.png" width="50px">](http://github.com/jonathanhoshi) &nbsp; &nbsp; &nbsp;
+[<img src="images/portfolio-icon.png" width="50px">](team/jonathanhoshi.md)
 
-* Role: Developer
-* Responsibilities: 
-  * Team Lead
-  * UI
+|   Role    | Responsibilities   |
+|:---------:|:-------------------|
+| Developer | Team Lead <br/> UI |
```
Comment (lowkaiwei98) [inline docs/AboutUs.md:32]: Should we reorder Jon to first if he taking team lead?

**Review by lowkaiwei98 — APPROVED** [review]

File: docs/AboutUs.md (hunk 1 above)
Comment (aweijun) [reply docs/AboutUs.md:21]: Yeap, I agree also, makes sense for the team lead to be at the top!

Comment (Ardentsoul) [reply docs/AboutUs.md:21]: Noted.

**Review by arcornior — APPROVED** [review]
LGTM. Nice design!

**Pull request merged**